self-serve-storage/pure-r/daaas_storage.R

```diff
@@ -10,31 +9,34 @@ get_bash_variable <- function (location, var) {
         intern = TRUE
     )
 }
-
 ### Just sets the environment variables.
 daaas_storage.__getClient__ <- function (storage_type) {
-
-    location = sprintf("/vault/secrets/minio-%s-tenant-1", storage_type)
-
-    MINIO_URL        = get_bash_variable(location, "MINIO_URL")
-    MINIO_ACCESS_KEY = get_bash_variable(location, "MINIO_ACCESS_KEY")
-    MINIO_SECRET_KEY = get_bash_variable(location, "MINIO_SECRET_KEY")
-
+    library(RJSONIO)
```

Comment:
I don't think `RJSONIO` is installed out-of-the-box in R notebooks, so this will fail unless someone installs it first. It looks like `jsonlite` is installed by default, and this package largely supersedes `RJSONIO` anyway.

Having a call to `library()` is bad form in a function. It can modify someone's search path, for example.